crates/bevy_pbr/src/lib.rs

```diff
@@ -192,6 +195,12 @@ impl Plugin for PbrPlugin {
             Shader::from_wgsl
         );
         load_internal_asset!(app, UTILS_HANDLE, "render/utils.wgsl", Shader::from_wgsl);
+        load_internal_asset!(
```

Comment:
I feel like this should be in `bevy_render`. Some people may want to use this in a custom impl that isn't using our PBR.